configure.ac

```diff
@@ -768,6 +771,39 @@ AS_IF([test x$with_lustre != xno],[
        MOTR_LINUX_SENDFILE
 ])
 
+#
+# Checking libfabric library----------------------------------------------- {{{1
+#
+
+AC_ARG_WITH([libfab],
+        [AS_HELP_STRING([--with-libfab=path],
+                        [set path to user installed libfabric headers and library])],
+        [LIBFABRIC=$with_libfab],
+        [LIBFABRIC=""]
+)
+
+AS_IF([test -z "$with_libfab" && ! test -f "/usr/include/rdma/fabric.h"],
```

Comment:
As I asked before, please confirm:

from now on, libfabric is a dependency lib. It must be installed on system.
It must be specified with --with-libfac or is installed in system lib dir.

Reply:
yes, it must be specified with --with-libfac or is installed in system lib dir.
@nikitadanilov  Is our understating correct ?

Reply:
If libfabric is not available, can we fall back to "sock"?

Or libfabric will be the only network transport when it is ready?

Reply:
No, this is not correct. motr build process should NOT assume that libfabric is installed. If it is installed, motr should detect it (in ./configure) and link with it. If it is not installed, we should use lnet and sock only. ./scripts/install-build-deps should install libfabric, but ./configure should not assume this. This would make the system less fragile.